dll/win32/shell32/stubs.cpp

```diff
@@ -451,26 +451,40 @@ SheSetCurDrive(INT iIndex)
     return 1;
 }
 
-/*
- * Unimplemented
- */
 EXTERN_C LPWSTR
 WINAPI
-SheRemoveQuotesW(LPWSTR lpInput)
+SheRemoveQuotesW(LPWSTR psz)
 {
-    FIXME("SheRemoveQuotesW() stub\n");
-    return NULL;
+    PWCHAR pch;
+
+    if (*psz == L'"')
+    {
+        for (pch = psz + 1; *pch && *pch != L'"'; ++pch)
+            *(pch - 1) = *pch;
```

Comment:
for-loop braces here too (I didn't want to repeat my previous comment)

Reply:
Fixed. Sorry.